8.0 base headers webkit

gurneyalex opened the pull request.

yvaucher committed:
reactivate base_headers_webkit

yvaucher committed:
improve descriptor

gurneyalex committed:
change ref to camptocamp_logo

and use a more neutral 'company_logo' name

gurneyalex commented:
updated version of #9 with @pedrobaeza requested fix

pedrobaeza commented on base_headers_webkit/base_headers_data.xml:
```diff
@@ -53,7 +53,7 @@
         <table class="header" style="border-bottom: 0px solid black; width: 100%">
             <tr>
-                <td>${helper.embed_logo_by_name('camptocamp_logo')|n}</td>
+                <td>${helper.embed_logo_by_name('company_logo')|n}</td>
```
I think this is not correct, because there is no `company_logo` imported. You have to put:

```
${helper.embed_image('', company.logo) | n}
```

lepistone commented:
I agree on removing the camptocamp logo. Still, it's on the `report_webkit` core module.

@nbessi, do you agree with @pedrobaeza's proposed solution?

Thanks!

nbessi commented:
@lepistone @gurneyalex please refer to my comment in https://github.com/OCA/webkit-tools/pull/11

nbessi commented:
I would prefer to override the image header definied in report_webkit module if possbile and try to make a patch in addons to change default header image in the meanwhile. But I version 8 it is less important.

pedrobaeza commented:
I insist in my option, because you would expect the same behaviour on Webkit reports than on standard QWeb ones, so change to the header to company logo (binary data in res.company record) is the most appropiated option. You cannot override rest of addons, because this data is in header, not in the mako definition.

nbessi commented:
@pedrobaeza well in this case the inconstancy is also true with default report_webkit header. If we use your solution we should modifiy default header the same way. By doing a patch in addons or by overriding them in base_header_webkit. I have no objection with that and it will unify the logo.

But that mean we have to ensure default logo dimentions will not break header. We must alse specify an height and a width by default in embeed_image

lepistone committed:
embed the company logo in the header

The idea is to remove the Camptocamp logo. We can't easily rename and
replace the logo file because it is in report_webkit in the core. So we
just take the company logo, which should make this consistent with other
reports.

lepistone commented:
I applied the change suggested py @pedrobaeza.

I marked that as "work in progress" because I want to port sale_order_webkit to test if that works fine.

lepistone commented:
blocked by https://github.com/odoo/odoo/issues/2334

gurneyalex committed:
synchronise with current head

yvaucher commented:
```
./base_headers_webkit/__openerp__.py:39:3: E127 continuation line over-indented for visual indent
./base_headers_webkit/__openerp__.py:46:3: E124 closing bracket does not match visual indentation
```

vrenaville commented:
This module also migrated to V8 in this PR https://github.com/OCA/webkit-tools/pull/24
